.circleci/config.yml

```diff
@@ -9,7 +9,9 @@ jobs:
       # documented at https://circleci.com/docs/2.0/circleci-images/
       # - image: circleci/mongo:3.4.4
       - image: trufflesuite/ganache-cli
-        command: ganache-cli -i 5777 -l 6700000
+        command: ganache-cli -i 5777 -l 6700000 -k istanbul
```

Comment:
@pmprete  I added the istanbul hardfork config to match the package.json script - is the gas limit needed?